packages/base-types/src/node/utils/command.ts

```diff
@@ -11,7 +11,8 @@ export enum CommandType {
 
 export interface TypedBaseCommand<Event extends BaseEvent = BaseEvent> extends BaseCommand {
   type: CommandType;
-  event: Event;
+  event?: Event;
```

Comment:
Can a command have event and on at the same time?

Reply:
No

Reply:
In that case would it require a lot of refactoring to create subtypes rather than having one type with optional 2 values, especially when one must be null when the other one exists?